examples/ball.js

```diff
@@ -31,11 +44,14 @@ window.initial_state = {
             height: 100,
             borderRadius: 50,
             zIndex: 1000,
-        },
+        }
     }
 }
-window.store = createStore(combineReducers({animations}))
-window.time = startAnimation(window.store, window.initial_state)
+window.store = createStore(combineReducers({
+    animations: animationsReducer,
+    ball: ballReducer,
```

Comment:
I'm coming around to liking Reducer in the name.  This map syntax is clear enough to make it obvious.